readme.md

```diff
@@ -384,6 +384,32 @@ Type: `object`
 
 Query items to add to the URL.
 
+### .pick(input, keysToPick)
+
+Returns a string with the `input` property without the keys you *did not enter* in the `keysToPick` property
+
```

Comment:
index.d.ts should be in sync with the readme docs-wise. That means using the same text as much as possible.